build/launch_training.py

```diff
@@ -97,14 +99,18 @@ def main():
                 peft_config=tune_config,
                 tracker_configs=tracker_config_args,
             )
-        except MemoryError:
+        except (MemoryError, OutOfMemoryError) as e:
             logging.error(traceback.format_exc())
-            write_termination_log("OOM error during training")
+            write_termination_log(f"OOM error during training. {e}")
             sys.exit(INTERNAL_ERROR_EXIT_CODE)
         except FileNotFoundError as e:
             logging.error(traceback.format_exc())
             write_termination_log("Unable to load file: {}".format(e))
             sys.exit(USER_ERROR_EXIT_CODE)
+        except HFValidationError as e:
+            logging.error(traceback.format_exc())
+            write_termination_log(f"Specified base model not found. Exception: {e}")
```

Comment:
I'm not sure if this is the only action that will throw `HFValidationError`, might be good to make more generic termination log, saying "HFValidationError: There may be a problem with loading the model"